test/test_integration.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
Can we use `test-integration.html` to align with above naming and existing gaia-components file naming convention?